zproject/prod_settings_template.py

```diff
@@ -168,6 +168,7 @@
 LOCAL_UPLOADS_DIR = "/home/zulip/uploads"
 #S3_AUTH_UPLOADS_BUCKET = ""
 #S3_AVATAR_BUCKET = ""
+#S3_REALM_ICONS_BUCKET = ""
```

Comment:
I've added the new bucket to avoid collision with avatar images and put them in a separate place. I can add folder realm_icons to `S3_AVATAR_BUCKET`.